src/cmd/alias.go

```diff
@@ -40,11 +40,10 @@ Note that this can conflict with other tools that also define additional Git com
 		}
 	},
 	PreRunE: func(cmd *cobra.Command, args []string) error {
-		err := validateArgsCount(args, 1)
-		if err != nil {
-			return err
-		}
-		return validateBooleanArgument(args[0])
+		return util.FirstError(
+			validateArgsCount(args, 1),
+			validateBooleanArgument(args[0]),
+		)
```

Comment:
I don't like this as it means these error checking functions get evaluated no matter what. In some cases we probably don't want later error checks to even happen if earlier ones fail. My first thought on how to do this is the following:
```go
return util.ErrorCheck(
  func() error { return validateArgsCount(args, 1) },
  func() error { return validateBooleanArgument(args[0]) },
)
```
This retains the type safety and I think is probably the best we can really do here.

Alternatively we could supply curried functions to make this look nicer:
```go
func validateArgsCountFunc(args []string, count int) (func() error) {
  return func() error { return validateArgsCount(args, 1) }
)

func validateBooleanArgumentFunc(arg string) (func() error) {
  return func() error { return validateBooleanArgument(args, 1) }
)

// Use

return util.ErrorCheck(
  validateArgsCountFunc(args, 1),
  validateBooleanArgumentFunc(args[0]),
)
```